src/user/user.reducer.js

```diff
@@ -63,6 +65,24 @@ export const userReducer = (state = initialState, action = {}) => {
         error: action.payload,
         isPendingOrgs: false,
       };
+    case GET_STAR_COUNT.PENDING:
+      return {
+        ...state,
+        starCount: ' ',
```

Comment:
Probably it is possible to remove, after all in `initialStore` already there is this key?

Reply:
@lex111 I have decided to remove it from `initialStore`, and only add to it when `GET_STAR_COUNT.PENDING` and `GET_AUTH_STAR_COUNT.PENDING` are respectively dispatched.

Reply:
Well, I would still put it in `initialStore`, and then it could only be added to `GET_STAR_COUNT.SUCCESS`, but I'm ok.